cmdstanpy/stanfit.py

```diff
@@ -698,17 +699,20 @@ def summary(
         cmd_path = os.path.join(
             cmdstan_path(), 'bin', 'stansummary' + EXTENSION
         )
-        tmp_csv_file = 'stansummary-{}-{}-chain-'.format(
-            self.runset._args.model_name, self.runset.chains
+        tmp_csv_file = 'stansummary-{}-'.format(
+            self.runset._args.model_name
         )
         tmp_csv_path = create_named_text_file(
-            dir=_TMPDIR, prefix=tmp_csv_file, suffix='.csv'
+            dir=_TMPDIR, prefix=tmp_csv_file, suffix='.csv', name_only=True
         )
+        csv_str = '--csv_filename={}'.format(tmp_csv_path)
+        if not cmdstan_version_at(2, 24):
```

Comment:
2.25?

Reply:
nope, 2.24 - 2.25 is still boost program options which is OK with this - that's why I didn't catch it sooner.